packages/spark/sodasql/dialects/spark_dialect.py

```diff
@@ -92,7 +95,8 @@ def sql_test_connection(self) -> bool:
 
     def sql_columns_metadata(self, table_name: str):
         with self.create_connection().cursor() as cursor:
-            cursor.execute(f"DESCRIBE TABLE {self.database}.{table_name}")
+            qualified_table_name = self.qualify_table_name(table_name)
+            cursor.execute(f"DESCRIBE TABLE {qualified_table_name}")
             return [(row[0], row[1], "YES") for row in cursor.fetchall()]
```

Comment:
What is the semantics behind "YES"? Is it some debugging value or is it passed somewhere else?

Reply:
This is something that we copied from other dialects (e.g.  [hive](https://github.com/sodadata/soda-sql/blob/aa11659c0d197c976e4011f9cc22ef81856083e8/packages/hive/sodasql/dialects/hive_dialect.py#L107)). "YES" means that the column is nullable.

We should create a data class to make more clear what is meant with this. I would do this outside of this PR